test/built-ins/Promise/any/resolve-from-resolve-reject-finally.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-promise.any
+description: >
+  Promise.any resolves with the first item that does not reject.
+flags: [async]
+features: [Promise.any, arrow-function]
+---*/
+
+let fulfillables = [
+  Promise.reject('a'),
+  new Promise((resolve, reject) => reject('b')),
+  Promise.all([Promise.reject('c')]),
+  Promise.resolve(Promise.reject('d').finally(v => v)),
```

Comment:
`Promise.reject('d').finally(v => v)` finally returns the original promise, and you already have a test for `catch(v => v)`, so probably just remove this test?